src/Api/Controllers/AuthController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Application.Services;

```

Comment:
Se for dar merge na PR anterior, atualizar da main após.
Senão adequar o namespace para Service.